backend/handlers/settings.go

```diff
@@ -11,6 +12,11 @@ import (
 	"gorm.io/gorm/clause"
 )
 
+type AccountError struct {
+	Error     string
```

Comment:
make this type error instead of string. That way you're not doing something unconventional like treating "" as no error